src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStepTest.java

```diff
@@ -724,6 +727,23 @@ private Object writeReplace() {
         j.waitForMessage("Timeout has been exceeded", b); // TODO assertLogContains fails unless a sleep is introduced; possible race condition in waitForCompletion
     }
 
+    @Issue("JENKINS-62014")
+    @Test public void envVarFilters() throws Exception {
+        EnvVarsFilterGlobalConfiguration.getAllActivatedGlobalRules().add(new RemoveSpecificVariablesFilter("FOO", "TODO: UNUSED"));
```

Comment:
I am pretty sure the `value` parameter in [the constructor](https://github.com/jenkinsci/generic-environment-filters-plugin/blob/05d03ec0f99eac8bb96dc5c6cad8fe52f0630b38/src/main/java/io/jenkins/plugins/generic_environment_filters/RemoveSpecificVariablesFilter.java#L43) for `RemoveSpecificVariablesFilter` is unused and could be removed. It's not used in [config.jelly](https://github.com/jenkinsci/generic-environment-filters-plugin/blob/05d03ec0f99eac8bb96dc5c6cad8fe52f0630b38/src/main/resources/io/jenkins/plugins/generic_environment_filters/RemoveSpecificVariablesFilter/config.jelly#L27) either from what I can see.

Reply:
Good point, copy & paste leftovers from https://github.com/jenkinsci/generic-environment-filters-plugin/blob/05d03ec0f99eac8bb96dc5c6cad8fe52f0630b38/src/main/java/io/jenkins/plugins/generic_environment_filters/VariableContributingFilter.java#L44; will fix in a future release.